tests/pkg_u8g2/Makefile

```diff
@@ -1,54 +1,59 @@
 APPLICATION = pkg_u8g2
 include ../Makefile.tests_common
 
-# These boards have not sufficient memory
-BOARD_BLACKLIST += chronos msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
-
 USEMODULE += xtimer
 
 USEPKG += u8g2
 
 # set default device parameters in case they are undefined
-TEST_U8G2_OUTPUT    ?= 1
-TEST_U8G2_DISPLAY   ?= u8g2_Setup_ssd1306_128x64_noname_1
+TEST_OUTPUT ?= 1
+
+TEST_I2C ?= 0
+TEST_SPI ?= 0
+
+TEST_ADDR ?= 0x78
 
-TEST_U8G2_I2C       ?= 0
-TEST_U8G2_SPI       ?= 0
+TEST_PIN_CS ?= GPIO_PIN\(0,0\)
+TEST_PIN_DC ?= GPIO_PIN\(0,0\)
+TEST_PIN_RESET ?= GPIO_PIN\(0,0\)
 
-TEST_U8G2_ADDR      ?= 0x78
+ifeq ($(TEST_OUTPUT),3)
```

Comment:
what do you think of merging this case with the same one a few lines below ? same for the I2C case.